src/components/input/input.stories.mdx

```diff
@@ -33,3 +41,64 @@ You can also add `c-input` to `<select>` elements, in which case you’ll get a
     })}
   </Story>
 </Preview>
+
+## Elastic textarea
+
+Add `js-elastic-textarea` to a `<textarea>` element\* to have it expand as the user types more than will fit.
+
+If the `rows` attribute is set, it will be used as the minimum height.
+
+<p>
+  <small>* see also the JavaScript usage notes below</small>
+</p>
+
+<Preview>
+  <Story name="Elastic Textarea" height="350px">
+    {() => {
+      // Use storybook hooks to trigger JS after story renders
+      // @see https://github.com/storybookjs/storybook/issues/7786
+      useEffect(() => {
+        // Select all elastic textarea elements, run create function on each
+        [...document.querySelectorAll('.js-elastic-textarea')].map(
+          (textarea) => {
+            return createElasticTextArea(textarea);
+          }
+        );
+      }, []);
+      return textDemo({
+        type: select(
+          'type',
+          ['text', 'email', 'search', 'date', 'textarea'],
+          'textarea'
+        ),
+        value: text('value', 'a\nb\nc\nd\ne'),
```

Comment:
Nit: Could we get some cloud four branded content here? Maybe a comment Jason or someone's written in a blog post comment thread?

Reply:
Also: is it confusing being able to change this either via knob or by typing in the textarea?

Reply:
> Could we get some cloud four branded content here?

Great idea, I'll do that.

> is it confusing being able to change this either via knob or by typing in the textarea?

Perhaps? I don't see a syntax to let me just _set_ a value without also making it a knob.

@tylersticka how do you feel about that?

Reply:
> Perhaps? I don't see a syntax to let me just set a value without also making it a knob.

I think that you would just omit the `text()` function: `value: 'a\nb\nc\nd\ne'`

Reply:
@spaceninja You should be able to just set a value. Just pass it along without the `text(...)` function.

Reply:
Ah, beauty! Thanks Tyler!